WazuhDB IT: Nightly report - Failed `test_wazuh_db_messages_agent`

| **Date** |  **Commit** | Build | 
| --- | --- |--- |
| 2022-04-06 | 99be0c9 | https://ci.wazuh.info/job/test_nightly/

## Tier 0


###  `test_wazuh_db_messages_agent`

- **Build**: https://ci.wazuh.info/job/Test_integration/24679/

- **Cases**
```
FAILED test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_agent[agent: Agents' CVEs table: vuln_cves]
```

- **Description**

```
    assert match, 'Failed test case stage {}: {}. Expected: {}. Response: {}' \
E   AssertionError: Failed test case stage 13: agent vuln_cves insert package with same CVE without checking if the package is present in sys_programs. Expected: ok {"action":"INSERT","status":"SUCCESS"}. Response: ok {"action":"INSERT","status":"ERROR"}
E   assert False
---------------------------- Captur
```

--- a/tests/integration/test_wazuh_db/data/agent/agent_messages.yaml
+++ b/tests/integration/test_wazuh_db/data/agent/agent_messages.yaml
@@ -103,14 +103,14 @@
                                         "type":"PACKAGE",
                                         "status":"VALID",
                                         "check_pkg_existence":false,
-                                        "severity":"-",
+                                        "severity":"Untriaged",
                                         "cvss2_score":0,
                                         "cvss3_score":0}'
     output: 'ok {"action":"INSERT","status":"SUCCESS"}'
     stage: "agent vuln_cves insert package with same CVE without checking if the package is present in sys_programs"
   -
     input: 'agent 000 sql SELECT * FROM vuln_cves WHERE name = "test_package2"'
-    output: 'ok [{"name":"test_package2","version":"3.0","architecture":"x86","cve":"CVE-2021-1001","detection_time":"*","severity":"-","cvss2_score":0,"cvss3_score":0,"reference":"99efe684b5ff4646b3c754de46cb6a9cbee9fbaa","type":"PACKAGE","status":"VALID"}]'
+    output: 'ok [{"name":"test_package2","version":"3.0","architecture":"x86","cve":"CVE-2021-1001","detection_time":"*","severity":"Untriaged","cvss2_score":0,"cvss3_score":0,"reference":"99efe684b5ff4646b3c754de46cb6a9cbee9fbaa","type":"PACKAGE","status":"VALID"}]'
     stage: "agent vuln_cves checking package insertion with same CVE"
     use_regex: "yes"
   -

